src/nodes/victron-virtual.js

```diff
@@ -82,6 +82,7 @@ const properties = {
     'Info/ChargeRequest': { type: 'i', format: (v) => v != null ? v : '', value: 0 },
     Soc: { type: 'd', min: 0, max: 100, format: (v) => v != null ? v.toFixed(0) + '%' : '', persist: 15 /* persist, but throttled to 15 seconds */ },
     Soh: { type: 'd', min: 0, max: 100, format: (v) => v != null ? v.toFixed(0) + '%' : '', persist: 60 /* persist, but throttled to 60 seconds */ },
+    TimeToGo: { type: 'd', max: 86400, format: (v) => v != null ? v.toFixed(0) + 's' : '' },
```

Comment:
Is this change intentional? It seems unrelated to the PR.

If intentional, mention it in the PR description.

Reply:
It was indeed intentional. Updated the PR.